extensions/binding/org.eclipse.smarthome.binding.yahooweather/src/main/java/org/eclipse/smarthome/binding/yahooweather/handler/YahooWeatherHandler.java

```diff
@@ -154,7 +156,9 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     private synchronized boolean updateWeatherData() {
         try {
-            weatherData = getWeatherData();
+            weatherData = getWeatherData(
+                    "https://query.yahooapis.com/v1/public/yql?format=json&q=SELECT%20*%20FROM%20weather.forecast%20WHERE%20u=%27c%27%20AND%20woeid%20=%20%27"
```

Comment:
the original query requests the complete weather information (select * from weather.forecast ...)

the new query I am using for the determination of the configstatus only requests only the location information (select location from weather.forecast) ... 

if you dont like this behaviour it is ok for me to remove the newly introduced query